### DISCUSSION TITLE
Add note taking 'templates' to the User Research section

### DISCUSSION THREAD

**Discussion by nhsbsa-sthil** [nhsbsa/nhsbsa-digital-playbook#216] [UCD]
## What

To help standardise the way we take notes in our user research sessions across the NHSBSA, we propose to create a number of best practice 'templates' for each User Researcher to use.

## Why

Currently each User Researcher has different templates when it comes to collecting notes for their sessions.

This can make it difficult for each note taker to understand what is expected of them when they move between different projects / services.

The hope is that by standardising them (and providing relevant guidance on how to take notes https://github.com/nhsbsa/nhsbsa-digital-playbook/discussions/217) we can ease this pain and ensure each User Researcher gets helpful and relevant notes.

Our ambition is to create a number of different templates based on each user research method, however as a starting point we propose to begin with the 2 of the most popular:

1. User Interviews
2. Usability Tests

**Comment by nhsbsa-sthil** [comment]
As a starting point, this is our proposed 'Usability Test' template:

[DDaT - Usability Test - TEMPLATE](https://nhsbsauk.sharepoint.com/:x:/t/UCD-DigitalProjects/EVFmB0cdHSFChcc56oKfYAEBrk-2u65hSwKxb3rA-2MhAA?e=Zv9Bg1)

**Reply by nhsbsa-thole** [reply]
I was just thinking about the process of filling this form out at the time (on the usability tab) and would there be both observations from the UR conducting the meeting and an additional observer? If there are multiple people inputting at the same time, it would make sense to have separate columns so they are not conflicting with eachother.

**Reply by nhsbsa-sthil** [reply]
That's a valid point @Tom-Leeds. Might be one to check with our UR's. Don't know if you have any thoughts @laurenmilor? 

I initially assumed that there would only be one 'official' note taker for the session - and the rest would just be observing. But if this isn't always the case we can amend things.

Would it be better having a document per note taker, or a single document for all notes?

**Reply by nhsbsa-thole** [reply]
I would vote for having a single document, it's conceivable that the number of files could quickly get out of hand and become unmanageable. Also it would cause duplication of the page titles/URL's being recorded, and further confusion if they don't match each other for whatever reason.

**Comment by nhsbsa-thole** [comment]
I have spoken to Jamie Thompson regarding the storing of staff names and dates of discussion for the purposes of creating an audit trail within the templates and he can see no issue with that. Just as long as we still never store personal information about those being interviewed/tested as usual.

**Comment by nhsbsa-sthil** [comment]
Agreed to store the templates once they're created in the newly created 'UCD DDaT - Projects' team in MS Teams, and then link to them via the playbook.

Here is the folder where they will be stored: 

https://nhsbsauk.sharepoint.com/:f:/t/UCD-DigitalProjects/EtBF1i3RlCNBhba-bcEI72EBCJY0sRuHJBvYgVpO6CsTiw?e=YMUaHB
